docs/README.md

```diff
@@ -26,34 +26,31 @@
 
-* `mode`: How Rio exposes the service mesh gateway. All HTTP requests to Rio services go through the gateway. There are three options:
+* `mode`: How Rio exposes the service mesh gateway. All HTTP requests to Rio services go through the gateway. There are two options:
 
 | Mode | Description |
 |------|-------------|
-| `ingress` | Rio will use existing ingress controller and ingress resource to expose gateway services. All the traffic will go through ingress. Starting v0.4.0 this is the default mode. |
 | `svclb` | Rio will use service loadbalancer to expose gateway services. | 
 | `hostport` | Rio will expose hostport for gateway services. |
 
+* `--mesh-mode`: Starting v0.5.0 Rio supports two service mesh mode: istio and linkerd. Linkerd will be default.
 * `http-port`: HTTP port gateway service will listen. You can only set the HTTP port if install mode is svclb or hostport. Default HTTP port for svclb and hostport mode is 9080. If install mode is ingress, HTTP port is determined by ingress and cannot be changed by Rio installer. Ingress controllers typically expose HTTP port 80, although some ingress controllers allow you to specify custom HTTP ports.
 * `https-port`: HTTPS port gateway service will listen. You can only set the HTTPS port if install mode is svclb or hostport. Default HTTPS port for svclb and hostport mode is 9443. If install mode is ingress, HTTPS port is determined by ingress and cannot be changed by Rio installer. Ingress controllers typically expose HTTPS port 443, although some ingress controllers allow you to specify custom HTTPS ports.
 * `ip-address`: Rio generates DNS domains that map to IP address of the gateway. Rio will attempt to detect gateway IP addresses automatically, and you can override by manually specify comma-separated IP addresses.
-* `service-cidr`: Rio will attempt to detect service CIDR for service mesh intercept traffic. You can override by manually specify a service CIDR.
 * `disable-features`: Specify feature to disable during install. Here are the available feature list.
 
 | Feature | Description |
 |----------|----------------|
 | autoscaling | Auto-scaling services based on QPS and requests load
 | build | Rio Build, from source code to deployment
-| grafana | Grafana Dashboard
-| istio | Service routing using Istio
-| kiali | Kiali Dashboard
+| gateway | Gateway service based on pilot and envoy
+| istio | Istio service mesh
+| linkerd | Linkerd service mesh
 | letsencrypt | Let's Encrypt
-| mixer | Istio Mixer telemetry
-| prometheus | Enable prometheus
 | rdns | Assign cluster a hostname from public Rancher DNS service
 
 * `httpproxy`: Specify HTTP_PROXY environment variable for Rio control plane. This is useful when Rio is installed behind an HTTP firewall.
-* `lite`: Disable all monitoring features including prometheus, mixer, grafana and kiali.
+* `lite`: Disable all monitoring features including prometheus, mixer, grafana and kiali. Only works in istio mesh mode.
```

Comment:
More of a question, linkerd also installs prometheus, so this flag with linkerd mode wouldn't disable the linkerd monitoring ?

Reply:
Yes, it won't disable linkerd monitoring. The original intention to use lite is that istio install is too large so we have to break it apart.